examples/cryptocurrency/src/lib.rs

```diff
@@ -172,6 +174,43 @@ transactions! {
     }
 }
 
+// // // // // // // // // // CONTRACT ERRORS // // // // // // // // // //
+
+/// Error codes emitted by `TxCreateWallet` and/or `TxTransfer` transactions during execution.
+#[derive(Debug, Fail)]
+#[repr(u8)]
+pub enum Error {
+    /// Wallet already exists.
+    ///
+    /// Can be emitted by `TxCreateWallet`.
+    #[fail(display = "Wallet already exists")]
+    WalletAlreadyExists = 0,
+
+    /// Sender doesn't exist.
+    ///
+    /// Can be emitted by `TxTransfer`.
+    #[fail(display = "Sender doesn't exist")]
+    SenderNotExist = 1,
```

Comment:
Maybe, use something like `AbsentSender` / `NonexistentSender` / `SenderNotFound`? `SenderNotExist` looks ungrammatic to me.

Reply:
Originally it was named `SenderDoesntExist", but the spell-checker wasn't happy.
Fixed.